pkg/storage/cacher_test.go

```diff
@@ -33,9 +33,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/watch"
-	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/testapi"
-	apitesting "k8s.io/kubernetes/pkg/api/testing"
+	"k8s.io/client-go/pkg/api"
```

Comment:
So you go the same route as I do in episode 3: use client-go's api code in tests. I hope this cyclic dependency does not bite us one day.